### PR TITLE
GCD of two Numbers in Javascript

### PR DESCRIPTION
**Pull request by tk315**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

gcd of two numbers code added in javascript

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#4691

### CODE REVIEW

**Comment by raksha009** [comment]
JavaScript is out of scope of our repo :)

File: JavaScript/math/gcd_of_two_no.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+A program to find Greatest Common Divisor(GCD) of two Numbers.
+GCD of two integers is the largest positive integer that divides each of the integers.
+*/
+
+//Take input from the user
+var number1=prompt("Enter 1st Number:");
+var number2=prompt("Enter 2nd Number:");
```
Comment (HarshCasper) [inline JavaScript/math/gcd_of_two_no.js]: Don't use prompt

File: JavaScript/math/gcd_of_two_no.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Output:
+5
+
+Input:
+4 7
+
+Output:
+1
+
+*/
```
Comment (HarshCasper) [inline JavaScript/math/gcd_of_two_no.js]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/math/gcd_of_two_no.js (hunk 1 above)
Comment (tk315) [reply JavaScript/math/gcd_of_two_no.js]: done

File: JavaScript/math/gcd_of_two_no.js (hunk 2 above)
Comment (tk315) [reply JavaScript/math/gcd_of_two_no.js]: done

**Comment by tk315** [comment]
Requested changes done
Kindly review @HarshCasper

File: JavaScript/math/gcd_of_two_no.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/*
+A program to find Greatest Common Divisor(GCD) of two Numbers.
+GCD of two integers is the largest positive integer that divides each of the integers.
+*/
+
+var number1 = 10;
+var number2 = 15;
```
Comment (HarshCasper) [inline JavaScript/math/gcd_of_two_no.js]: Use Dynamic Input

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/math/gcd_of_two_no.js (hunk 3 above)
Comment (tk315) [reply JavaScript/math/gcd_of_two_no.js]: what should i use prompt or something else

Comment (HarshCasper) [reply JavaScript/math/gcd_of_two_no.js]: Take the input from the User.

Comment (tk315) [reply JavaScript/math/gcd_of_two_no.js]: Done

**Comment by tk315** [comment]
Requested changes done
Kindly review @HarshCasper @ankitaggarwal23 @gat786 @iamrajiv

**Review by gat786 — APPROVED** [review]

**Review by iamrajiv — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**